angular-ngrx-scss/src/app/shared/components/repo-controls/repo-controls.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { provideMockStore } from '@ngrx/store/testing';
+import { RepoControlsComponent } from './repo-controls.component';
+
+describe('RepoControlsComponent', () => {
```

Comment:
blocker(tests): We should add tests for this component.

Reply:
I'm working on it, should be done tomorrow morning for you.